File: Bai_61/main.cpp
#include <iostream>
using namespace std;

const int MOD = 2018;

long long binomial_coefficient(long long n, long long k) {
    if(k > n - k) k = n - k;
    long long res = 1;
    for(long long i = 0; i < k; i++){
        res *= (n - i);
        res %= MOD; // Sử dụng phép toán modulo
        res /= (i + 1);
    }
    return res % MOD;
}

int main() {
    long long n, k;
    freopen("BAI1.INP", "r", stdin);
    cin >> n >> k;
    freopen("BAI1.OUT", "w", stdout);
    cout << binomial_coefficient(n, k);
    return 0;
}

File: Bai_61/main.py
def binomial_coefficient(n, k):
    if k > n - k:
        k = n - k
    res = 1
    for i in range(k):
        res *= (n - i)
        res //= (i + 1)
    return res % 2018

with open("BAI1.INP", "r") as f_in:
    n, k = map(int, f_in.readline().split())

result = binomial_coefficient(n, k)

with open("BAI1.OUT", "w") as f_out:
    f_out.write(str(result))